lib/acpi/Kconfig

```diff
@@ -14,12 +14,6 @@ module = ACPI
 module-str = acpi
 source "subsys/logging/Kconfig.template.log_config"
 
-config ACPI_PRT_BUS_NAME
```

Comment:
You need to remove this config from all boards configurations

Reply:
updated